sky/skylet/job_lib.py

```diff
@@ -115,6 +116,17 @@ def __lt__(self, other):
         return list(JobStatus).index(self) < list(JobStatus).index(other)
 
 
+_JOB_STATUS_TO_COLOR = {
+    JobStatus.INIT: colorama.Fore.BLUE,
+    JobStatus.SETTING_UP: colorama.Fore.BLUE,
+    JobStatus.PENDING: colorama.Fore.BLUE,
+    JobStatus.RUNNING: colorama.Fore.GREEN,
+    JobStatus.SUCCEEDED: colorama.Fore.GREEN,
+    JobStatus.FAILED: colorama.Fore.RED,
+    JobStatus.FAILED_SETUP: colorama.Fore.RED,
+    JobStatus.CANCELLED: colorama.Fore.YELLOW,
+}
+
```

Comment:
It would be nice to place this color mapping inside the `JobStatus` class, e.g. we can have a method `def colored_str(self):`